### DISCUSSION TITLE
能否以脚本的方式解决全屏播放界面无法自动隐藏的问题

### DISCUSSION THREAD

**Discussion by q56535** [the1812/Bilibili-Evolved#3271] [功能建议 / Ideas]
b站的视频在进入全屏时不会自动隐藏播放界面，必须要移动指针才会自动隐藏，而有人可能只会点一次全屏而不会有意或无意移动鼠标，导致播放界面不自动消失影响观感，因此希望可以以脚本的方式解决
[视频演示](https://user-images.githubusercontent.com/26663837/164865846-fcb5b226-e0f3-443a-95dd-3c169fbf81be.mp4)

**Comment by the1812** [comment]
是不是可以用双击全屏来解决, 这样隐藏就不是由鼠标移动来控制的了

**Reply by q56535** [reply]
双击全屏确实可以自动隐藏，感谢答复

**Reply by q56535** [reply]
在我用了一小段时间后发现，我根本没有意识去双击全屏，就如同没有这个功能一样
